ts/Dashboards/Plugins/HighchartsComponent/HighchartsSyncHandlers.ts

```diff
@@ -245,13 +245,9 @@ const configs: {
 
                                 // Prefer a series that's in a related table,
                                 // but allow for other data
-                                const seriesInTable = axis.series
-                                    .filter((series): boolean =>
-                                        table.hasColumns([series.name]));
-
-                                const [series] = seriesInTable.length ?
-                                    seriesInTable :
-                                    axis.series;
+                                const series = component.seriesFromConnector.length > 0 ?
```

Comment:
Will it make sens to have an early return if this is false `component.seriesFromConnector.length > 0`?

Reply:
I would leave it as it is now, because I assume it will be possible to synchronize the axis extremes in the case of charts that have a connector but do not have a series connected to it. I think this is a case of more investigation and redesign.